lib/core/lib/ros/core.rb

```diff
@@ -93,7 +93,7 @@ def decode
 
     def alg; Settings.jwt.alg end
 
-    def encryption_key; Settings.jwt.encryption_key end
+    def encryption_key; Settings.jwt.encryption_key || '' end
```

Comment:
I am not sure preventing this from failing is actually a good idea. This would mean that we could potentially be generating the tokens without any encryption key without anyone noticing. Imagine for some reason we forget to set the key in the settings, then all tokens would be generated without the enc key, which means that anyone could generate the same tokens and have a security flaw. In this case, I think we shouldnt generate the token if the enc key is not set and rather raise a proper error.